app/pages/events/event_list/event_calendar.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+restrict_access();
+restrict_feature(Feature::Calendar);
+
+$year = $_GET['year'] ?? date('Y');
+$month = $_GET['month'] ?? date('m');
+$firstDayOfMonth = new DateTime("$year-$month-01");
+$startDate = (clone $firstDayOfMonth)->modify("Monday this week");
```

Comment:
What is the clone here for ?

Reply:
It's to make a new date object with same date as the previous